docs/2_app_operators/1_packaging_an_app.md

```diff
@@ -0,0 +1,185 @@
+maintainers:
+- name: Daniel Suskin
+  email: daniel@upbound.io
+
+# Owner names and emails.
+owners:
+- name: Daniel Suskin
+  email: daniel@upbound.io
+
+# Human readable company name.
+company: Upbound
+
+# Type of package: Provider, Stack, or Application
+packageType: Application
+
+# Keywords that describe this application and help search indexing
+keywords:
+- "samples"
+- "examples"
+- "tutorials"
+- "wordpress"
+
+# Links to more information about the application (about page, source code, etc.)
+website: "https://upbound.io"
+source: "https://github.com/crossplane/app-wordpress"
+
+# RBAC Roles will be generated permitting this stack to use all verbs on all
+# resources in the groups listed below.
+permissionScope: Namespaced
+dependsOn:
+- crd: "kubernetesclusters.compute.crossplane.io/v1alpha1"
+- crd: "mysqlinstances.database.crossplane.io/v1alpha1"
+- crd: "kubernetesapplications.workload.crossplane.io/v1alpha1"
+- crd: "kubernetesapplicationresources.workload.crossplane.io/v1alpha1"
+
+# License SPDX name: https://spdx.org/licenses/
+license: Apache-2.0
+```
+
+**behavior.yaml** `[required]`
+
+While the `app.yaml` is responsible for metadata, that `behavior.yaml` is
+responsible for operations. It is where you tell Crossplane how to create
+resources in the cluster when an instance of the [CustomResourceDefinition]()
+that represents your application is created. Take a look at the Wordpress
+`behavior.yaml` for reference:
+
+```yaml
+source:
+  path: "helm-chart" # where the configuration data exists in Docker container
+crd:
+  kind: WordpressInstance # the kind of the CustomResourceDefinition
+  apiVersion: wordpress.apps.crossplane.io/v1alpha1 # the apiVersion of the CustomResourceDefinition
+engine:
+  type: helm3 # the configuration engine to be used (helm3 and kustomize are valid options)
+```
+
+**icon.svg**
+
+The `icon.svg` file is a logo for your application.
+
+**resources/** `[required]`
+
+The `resources/` directory contains the CustomResourceDefinition (CRD) that
+Crossplane watches to apply the configuration data you supply. For the Wordpress
+application, this is `wordpress.apps.crossplane.io_wordpressinstances.crd.yaml`.
+CRDs can be generated from `go` code using projects like
+[controller-tools](https://github.com/kubernetes-sigs/controller-tools), or can
+be written by hand. 
+
+You can also supply metadata files for your CRD, which can be parsed by a user
+interface. The files must match the name of the CRD kind for your application:
+
+- `<your-kind>.icon.svg`: an image to be displayed for your application CRD
+- `<your-kind>.resource.yaml`: a description of your application CRD
+- `<your-kind>.ui-schema.yaml`: the configurable fields on your CRD that you
+  wish to be displayed in a UI
+
+Crossplane will take these files and apply them as
+[annotations](https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/)
+on the installed application. They can then be parsed by a user interface.
```

Comment:
technically the `installed CRD resources`, but "application" could generically cover that

Reply:
Are the annotations applied on a `Stack` object?